index.bs

```diff
@@ -1869,6 +1867,111 @@ To <dfn>serialize [=event-level report/attribution destinations=]</dfn> |destina
 1. If |destinationStrings|'s [=set/size=] is equal to 1, return |destinationStrings|[0].
 1. Return |destinationStrings|.
 
+<h3 id="encode-integer">Encode an unsigned k-bit integer</h3>
+
+To <dfn>encode an unsigned k-bit integer</dfn>, represent it as a big-endian [=byte sequence=]
+of length k / 8, left padding with zero as necessary.
+
+<h3 id="obtain-aggregatable-report-debug-mode">Obtaining an aggregatable report's debug mode</h3>
+
+An [=aggregatable report=] |report|'s <dfn for="aggregatable report">debug mode</dfn> is the result
+of running the following steps:
+
+1. If |report|'s [=aggregatable report/source debug key=] is null, return <strong>disabled</strong>.
+1. If |report|'s [=aggregatable report/trigger debug key=] is null, return <strong>disabled</strong>.
+1. Return <strong>enabled</strong>.
+
+<h3 id="obtain-aggregatable-report-shared-info">Obtaining an aggregatable report's shared info</h3>
+
+An [=aggregatable report=] |report|'s <dfn for="aggregatable report">shared info</dfn> is the result
+of running the following steps:
+
+1. Let |reportingOrigin| be |report|'s [=aggregatable report/reporting endpoint=].
+1. [=Assert=]: |reportingOrigin| is a [=potentially trustworthy origin=].
```

Comment:
I don't think we need this assertion anymore, as the reporting endpoint is a suitable origin.